.github/workflows/build-and-test-external.yml

```
@@ -0,0 +1,54 @@
+name: Build and Test External
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name != 'develop' && github.ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+
+jobs:
+  determine-affected:
+    name: "Turbo Affected"
+    if: ${{contains(needs.determine-affected.outputs.paths, 'ledger-live-desktop') && github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/turbo-affected-reusable.yml@develop
```

if you tag `@develop` you need to specify a `owner/repo` syntax
```suggestion
    uses: ledgerhq/ledger-live/.github/workflows/turbo-affected-reusable.yml@develop
```